sympy/core/numbers.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import print_function, division
 
 import decimal
+from decimal import *
```

Comment:
Do not use `import *`. `decimal` is already imported in the line above this. You could just use that.